src/System.Linq/src/System/Linq/Errors.cs

```diff
@@ -6,18 +6,18 @@ namespace System.Linq
 {
     internal static class Error
     {
-        internal static Exception ArgumentNull(string s) { return new ArgumentNullException(s); }
+        internal static Exception ArgumentNull(string s) => new ArgumentNullException(s);
```

Comment:
Again a question rather than a comment: Is this now preferred for one-expression methods?